packages/genesys-spark-components/src/components/stable/gux-listbox-multi/gux-listbox-multi.tsx

```diff
@@ -240,9 +241,27 @@ export class GuxListboxMulti {
 
   // get list of listbox option elements
   private setListboxOptions(): void {
-    this.listboxOptions = (
-      Array.from(this.root.children) as HTMLGuxOptionMultiElement[]
-    ).filter(element => element.tagName === 'GUX-OPTION-MULTI');
+    const options: HTMLGuxOptionMultiElement[] = [];
+    const listChildren = Array.from(this.root.children);
+    listChildren.map(child => {
+      if (isOptionGroup(child)) {
+        const childOptions = Array.from(
+          child.children
+        ) as HTMLGuxOptionMultiElement[];
+        return options.push(...childOptions);
+      }
+      return options.push(child as HTMLGuxOptionMultiElement);
+    });
+
+    const filteredOptions: HTMLGuxOptionMultiElement[] = options.filter(
```

Comment:
I move move this filter so that it is used to guard what is pushed into the options array instead of filtering the options array after adding everything.

Also is this the code that would be simplified if we used a `label` property instead of a named slot?